src/EventExport/CalendarSummary/CalendarSummaryWithFormatterRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\EventExport\CalendarSummary;
+
+use CultuurNet\CalendarSummaryV3\CalendarHTMLFormatter;
+use CultuurNet\CalendarSummaryV3\CalendarPlainTextFormatter;
+use CultuurNet\CalendarSummaryV3\Offer\Offer;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+
+final class CalendarSummaryWithFormatterRepository implements CalendarSummaryRepositoryInterface
+{
+    private DocumentRepository $repository;
```

Comment:
I think the name can be more explicit, this is an `eventRepository` or `eventJsonRepository`